modules/serverless-rn-1-7-0.adoc

```diff
@@ -0,0 +1,41 @@
+
+[id="serverless-rn-1-7-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} 1.7.0
+
+[id="new-features-1-7-0_{context}"]
+== New features
+* {ServerlessProductName} 1.7.0 is now Generally Available (GA) on {product-title} 4.3 and newer versions. In previous versions, {ServerlessProductName} was a Technology Preview.
+* {ServerlessProductName} now uses Knative Serving 0.13.2.
+* {ServerlessProductName} now uses Knative Serving Operator 0.13.2.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.13.2.
+* Knative `kn` CLI downloads now support disconnected, or restricted network installations.
+* Knative `kn` CLI libraries are now signed by Red Hat.
+* Knative Eventing is now available as a Technology Preview with {ServerlessProductName}. {ServerlessProductName} uses Knative Eventing 0.13.2.
+
+[IMPORTANT]
+====
+Before upgrading to the latest Serverless release, you must remove the community Knative Eventing operator if you have previously installed it. Having the Knative Eventing operator installed will prevent you from being able to install the latest Technology Preview version of Knative Eventing that is included with {ServerlessProductName} 1.7.0.
+====
+
+* High availability (HA) is now enabled by default for the `autoscaler-hpa`, `controller`, `activator` , `kourier-control`, and `kourier-gateway` components.
++
+If you have installed a previous version of {ServerlessProductName}, after the KnativeServing custom resource (CR) is updated, the deployment will default to a HA configuration with `KnativeServing.spec.high-availability.replicas = 2`.
++
+You can disable HA for these components by completing the procedure in the _Configuring high availability components_ documentation.
+* {ServerlessProductName} now supports the `trustedCA` setting in {product-title}'s cluster-wide proxy, and is now fully compatible with {product-title}'s proxy settings.
+* {ServerlessProductName} now supports HTTPS using the wildcard certificate that is registered for {product-title} routes. For more information on `http` and `https` on Knative Serving, see the documentation on _Verifying your serverless application deployment_.
+
+[id="fixed-issues-1-7-0_{context}"]
+== Fixed issues
+* In previous versions, requesting KnativeServing custom resources (CRs) without specifying an API group, for example, `oc get knativeserving -n knative-serving`, occasionally caused errors. This issue is fixed in {ServerlessProductName} 1.7.0.
+
+[id="known-issues-1-7-0_{context}"]
+== Known issues
+* When upgrading from {ServerlessProductName} 1.6.0 to 1.7.0, support for HTTPS requires a change to the format of routes. This causes Knative services created on {ServerlessProductName} 1.6.0 to stop working.
+* If you are installing {ServerlessProductName} on an Azure cluster, it is possible that the `imc-dispatcher` pod may not start. This is due to the pod's default `resources` settings. As a work-around, you can remove the `resources` settings.
```

Comment:
This is specific to Eventing, maybe we should call out "If you are installing {ServerlessProductName} and use the Knative Eventing APIs"?